catkin_ws/src/robocar_simulation/launch/robocar_sim.launch

```diff
@@ -4,21 +4,21 @@
   <env name="GAZEBO_RESOURCE_PATH" value="$(find robocar_world)"/>
   
   <arg name="world_name" default="$(find robocar_world)/worlds/empty.world"/>
-  <arg name="paused" default="false"/>
+  <arg name="paused" default="true"/>
```

Comment:
why paused?

Reply:
I thought that we must wait till world and car model in gazebo are fully loaded, all controllers are loaded, all messages are correctly sent. E.g. almost always first images from camera in simulator are empty and it has bad effect on lane detection.